static/app/components/clippedBox.tsx

```diff
@@ -199,7 +209,11 @@ function ClippedBox(props: ClippedBoxProps) {
   );
 
   return (
-    <Wrapper ref={onWrapperRef} className={props.className}>
+    <Wrapper
+      ref={onWrapperRef}
+      className={props.className}
+      onTransitionEnd={() => clearMaxHeight(wrapperRef)}
```

Comment:
Can we bind to this event from the component and cleanup the listener after the reveal? This way we would avoid this possibly firing multiple times for no reason + we avoid an unstable fn ref

Reply:
Good feedback! That does seem like a safer approach.

Added here, LMK if this is what you were thinking: [`49f71d3` (#62015)](https://github.com/getsentry/sentry/pull/62015/commits/49f71d3ab8d131cc59990321f222815240191194)

While making that change I realized that the height wasn't correct when the wrapper had padding, so I added that as well.